src/components/anchor/nz-anchor.component.ts

```diff
@@ -0,0 +1,157 @@
+import {
+  Component,
+  ViewEncapsulation,
+  OnInit,
+  Input,
+  EventEmitter,
+  Output,
+  Renderer2,
+  OnDestroy,
+  ViewChild,
+  ElementRef,
+  Inject
+} from '@angular/core';
+import { DOCUMENT } from '@angular/platform-browser';
+import { Subscription } from 'rxjs/Subscription';
+import { Observable } from 'rxjs/Rx';

```

Comment:
Should only import specific type as suggested in https://github.com/NG-ZORRO/ng-zorro-antd/pull/29